cmake/SetupBlaze.cmake

```diff
@@ -57,6 +57,44 @@ if(SLEEF_FOUND)
   set(_BLAZE_USE_SLEEF 1)
 endif()
 
+# If BLAZE_USE_STRONG_INLINE=1, Blaze will use this keyword to increase the
```

Comment:
Can you not use the syntax `option(USE_SLEEF "Use Sleef to add more vectorized instructions." OFF)` to control the inlining? This I think does most of the checking you have, except the error handling.

Reply:
I just used if/else because I'm not familiar with cmake but I just looked it up this function's documentation. You're proposing to use this to set the default of these two flags to `1`?

Reply:
Yea, that's right :)

Reply:
Alright, gave it a shot and updated it to use `option` in the fixup!